json_spec/input/package_info.json

```diff
@@ -53,5 +53,10 @@
     "Version": "",
     "JSON": "",
     "Vendor": ""
+  },
+  "GPU_FW_IMAGE_EMPTY_METADATA": {
+    "CorruptComponentIdentifier": "",
+    "MetadataSizeBytes": 1024,
```

Comment:
Lets set default to 4096 which matches latest PLDM bundles.  We can fix this in upcoming change.

Reply:
Changed the default to 4096 bytes in the latest commit.